### DISCUSSION TITLE
플레이어의 국적을 서버에서 알 수 있는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#974] [5. Q&A]
혹시 알 수 있다면 국적에 맞게 언어를 반영해서 서비스 하려고 합니다.
알려주세요🙏

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, UnityEngine.Applicaiton.systemLanguage 사용이 가능합니다.
참고로 제페토 앱 언어 설정은 디바이스의 언어 설정을 따라갑니다.

systemLanguage를 통해서 언어를 반영하도록 개발하시는 것을 추천 드립니다.
감사합니다.
